### DISCUSSION TITLE
Basis texture in glTF

### DISCUSSION THREAD

**Discussion by goldenratio** [jnsmalm/pixi3d#132] [Q&A]
Hey, wondering if it's possible to use basis texture  in .gltf and .glb
https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_texture_basisu/README.md

**Comment by jnsmalm** [comment]
Hey!  I think it might be possible but I have not tested it myself. PixiJS have a plugin for this: https://api.pixijs.io/@pixi/basis/PIXI/BasisLoader.html.  As long as it's a Pixi texture it should work.
